OWNERS_ALIASES

```diff
@@ -17,6 +17,7 @@ aliases:
     - enj
     - liggitt
     - mikedanese
+    - ritazh
     - tallclair
```

Comment:
Remove?

Reply:
@fejta Looks like https://github.com/kubernetes/org/pull/2894/files#diff-4985b733677adf9dda6b5187397d4700868248ef646d64aecfb66c1ced575499L20 has addressed this